src/main/java/me/spartacus04/jext/listeners/JukeboxClickEvent.kt

```diff
@@ -31,18 +31,10 @@ internal class JukeboxClickEvent : JextListener() {
 
     private fun defaultBehaviour(event: PlayerInteractEvent, block: Block) {
         if(INTEGRATIONS.hasJukeboxAccess(event.player, block)==false) {
-            event.player.sendMessage("Missing flag")
             event.isCancelled = true
             return
         }
-        val t = if(event.player.hasPermission("jext.notifyupdate")) "true" else "false"
-        event.player.sendMessage(t);
-        event.player.sendMessage(if(event.player.hasPermission("jext.usejukebox")) "true" else "false")
         if(!(event.player.hasPermission("jext.usejukebox")||event.player.isOp)){
-            for(perm in event.player.effectivePermissions){
-                event.player.sendMessage(perm.permission)
-            }
-            event.player.sendMessage("Missing permission")
             event.isCancelled = true
             return
         }
```

Comment:
This check should be moved to the `OnJukeboxInteract` method, you can also remove the `|| event.player.isOp` since it should be managed by a permission manager and not the plugin